src/utils/__tests__/kbn_field_types.js

```diff
@@ -0,0 +1,51 @@
+import expect from 'expect.js';
+
+import {
+  KbnFieldType,
+  getKbnFieldType,
+  castEsToKbnFieldTypeName,
+  castEsToKbnFieldType,
+  getEsTypes
+} from '../kbn_field_types';
+
+describe('utils/kbn_field_types', () => {
+  describe('getKbnFieldType()', () => {
+    it('returns a KbnFieldType instance by name', () => {
+      expect(getKbnFieldType('string')).to.be.a(KbnFieldType);
+    });
+
+    it('returns undefined for invalid name', () => {
+      expect(getKbnFieldType(Math.random())).to.be(undefined);
```

Comment:
I generally hesitate to use static strings for "invalid" values, so I wanted to use a little randomness and got lazy.